t/daemon/formats/jats.xml

```diff
@@ -138,12 +138,12 @@ Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
           <list-item id="S1.I2.i2">
             <p id="S1.I2.i2.p1">two</p>
             <p>
-              <list list-type="order" id="S1.I2.I1">
-                <list-item id="S1.I2.i2.i1">
-                  <p id="S1.I2.i2.i1.p1">one</p>
+              <list list-type="order" id="S1.I2.i2.I2">
+                <list-item id="S1.I2.i2.I2.i1">
+                  <p id="S1.I2.i2.I2.i1.p1">one</p>
```

Comment:
`S1.I2.i2.I2.i1.p1` is a fascinating construction, but certainly quite usable. As long as we don't have to present it to any ID-skeptical committees, seems good to me! An verbose structural ID is definitely better than a clashing ID with a patch.